scripts/merge_poll_discovery_gap.py

```
@@ -1,5 +1,6 @@
-#!/usr/bin/env python3
-"""Merge discovery-enabled BACnet points missing from points.csv (poll driver list)."""
+#!/usr/bin/env bash
```

Ruff (0.15.17)

[warning] Shebang should contain `python`, `pytest`, or `uv run`

(EXE003)